File: app/src/main/java/com/example/parking/data/models/Car.java
package com.example.parking.data.models;

public class Car {
    private String id;
    private String model;
    private String num;

    public Car(String id, String model, String num) {
        this.id = id;
        this.model = model;
        this.num = num;
    }

    public String getId() {
        return id;
    }

    public String getModel() {
        return model;
    }

    public String getNum() {
        return num;
    }

}

File: app/src/main/java/com/example/parking/data/models/Car.kt
package com.example.parking.data.models

import java.util.*
import kotlin.collections.HashMap


class Car (
    val id: UUID = UUID.fromString("0f29717c-37e4-4a93-9165-6baacae64e98"),
    var model: String,
    var length: Int = 400,
    var wight: Int = 160,
    var registryNumber: String) {

    fun toHashMap(withID: Boolean = false): HashMap<String, Any> {
        return if (withID) hashMapOf(
            "id" to id.toString(),
            "model" to model,
            "registryNumber" to registryNumber,
            "wight" to wight,
            "length" to length
        ) else
            hashMapOf(
                "model" to model,
                "registryNumber" to registryNumber,
                "wight" to wight,
                "length" to length
            )
    }
}